config.py.example

```diff
@@ -101,3 +102,20 @@ else:
 # your own access token. See
 # help.github.com/articles/creating-an-access-token-for-command-line-use
 OAUTH_TOKEN = os.environ.get('OAUTH_TOKEN') or ""
+
+# Status categories used in the project
+# 'new', 'needsdiagnosis', 'needscontact', 'contactready' , 'sitewait', 'close'
+# Creating the model
+Category = namedtuple('Category', ['name', 'dataAttribute', 'label'])
+CATEGORIES = []
+cat_labels = [('new', 'new', 'New Issues'),
+              ('needsDiagnosis', 'needsdiagnosis', 'Needs Diagnosis'),
+              ('needsContact', 'needscontact', 'Needs Contact'),
+              ('ready', 'contactready', 'Ready for Outreach'),
+              ('sitewait', 'sitewait', 'Site Contacted'),
+              ('close', 'closed', 'Closed')]
```

Comment:
status of the bug != other type of labels. These are really status.
the `cat_labels` is probably not appropriate and should be `status_labels`.

Note also that this was initially designed to answer a particular need for @magsout in its templating use, with the idea of extending to other parts of the code in a second strike.

This can evolve, but we have to be careful, this is used by @magsout